### DISCUSSION TITLE
Need serenityjs capabilities deck ppt

### DISCUSSION THREAD

**Discussion by harivsudhan** [serenity-js/serenity-js#1648] [How to... ?]
@jan-molak 
Hi Jan- Great morning! I have implemented serenityjs protractor with Rest  in typescript, I have been requested to do a presentation on the Serenityjs capabilities. Could you please help if you have any Deck on this so that i can do a presentation to my team.

Thanks in advance!

**Comment by jan-molak** [comment]
Hey @harivsudhan! Are there any specific assets you need for the presentation?

You can find the [high-level value proposition](https://github.com/serenity-js/serenity-js/blob/main/documentation/serenity-js.org/docs/index.mdx) of the framework in the docs that will accompany Serenity/JS v3.
You can find Serenity/JS logos [here](https://github.com/serenity-js/serenity-js/tree/main/documentation/serenity-js.org/static/images).

Let me know if you need anything else.

BTW: it seems like it would be useful to have presentation materials organised in one place so that they're easier to find for Serenity/JS champions like yourself. Let's see what materials we gather as part of this exercise, and I'll package them nicely for the community afterwards.

**Comment by harivsudhang** [comment]
@jan-molak Hi Jan - I am looking for some power-point presentations about the architecture and other capabilities like ( Reusability, reporting, CI/CD, Web/API ). Is that possible?

**Comment by jan-molak** [comment]
Hello @harivsudhan! The new website with the list of [high-level capabilities](https://serenity-js.org/handbook/) is now live.

I'm planning to start working on preparing some materials to help Serenity/JS Community promote the framework soon, so stay tuned!

**Comment by viper3400** [comment]
Hi @harivsudhan  / @harivsudhang  Jan held a talk, recently: "Creating Faster and More Reliable Web Tests with Blended Testing" at SeleniumConf Chicago. It's now available on YouTube.

https://www.youtube.com/watch?v=pan6yyXgNbw&list=PLRdSclUtJDYXDEsWI0vwBmJxW17NgsaAk&index=12

He also provided the slides: https://speakerdeck.com/janmolak/creating-faster-and-more-reliable-web-tests-with-blended-testing

Is this something that will help you?
